### PR TITLE
Minicarousel

### PR DESCRIPTION
**Pull request by claire123c**
- Added up and down buttons to switch thumbnail images
- Added tests for both up and down buttons

### CODE REVIEW

File: client/src/components/overview/ImgGallery/Large.jsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ function Large(props) {
       useCurrentImg(photos[currentImgIndex + 1]);
     }
   };
-  const onClickThumb = (current, i) => {
+  const onClickThu = (current, i) => {
```
Comment (johnamielyasis) [inline client/src/components/overview/ImgGallery/Large.jsx:64]: Not sure about this var name change; I feel like onClickThumb was more descriptive

**Review by johnamielyasis — APPROVED** [review]
Some nitpicking but otherwise lgtm

File: client/src/components/overview/ImgGallery/Large.jsx (hunk 1 above)
Comment (claire123c) [reply client/src/components/overview/ImgGallery/Large.jsx:64]: Thanks John! I agree, I changed it for airbnb styling purposes. The name was too long when I was trying to pass it on as props.

Comment (phil-ho) [reply client/src/components/overview/ImgGallery/Large.jsx:64]: airbnb said your name was too long? or the line was too long?

Comment (claire123c) [reply client/src/components/overview/ImgGallery/Large.jsx:64]: Oops, yeah, the line was too long.

**Pull request merged**